File: framework/module_service/src/main/java/com/xxl/hello/service/upload/impl/TencentUploadServiceImpl.java
package com.xxl.hello.service.upload.impl;

import android.app.Application;
import android.util.Log;

import androidx.annotation.NonNull;

import com.xxl.hello.service.data.repository.DataRepositoryKit;
import com.xxl.hello.service.upload.api.UploadService;

/**
 * 腾讯云上传服务实现类
 *
 * @author xxl.
 * @date 2022/5/28.
 */
public class TencentUploadServiceImpl implements UploadService {

    //region: 成员变量

    //endregion

    //region: 构造函数

    public TencentUploadServiceImpl(@NonNull final Application application,
                                    @NonNull final DataRepositoryKit dataRepositoryKit) {

    }


    //endregion

    //region: 页面生命周期

    /**
     * 开始上传
     *
     * @param key
     */
    @Override
    public void onStart(@NonNull String key) {
        Log.e("aaa", "onStart: 我是腾讯云上传" + key);
    }

    /**
     * 上传完成
     *
     * @param domain
     * @param key
     */
    @Override
    public void onComplete(@NonNull String domain, @NonNull String key) {

    }

    //endregion

    //region: 内部辅助方法

    //endregion

}

File: framework/module_service/src/main/java/com/xxl/hello/service/upload/impl/TencentUploadServiceImpl.kt
package com.xxl.hello.service.upload.impl

import android.app.Application
import android.util.Log
import com.xxl.core.service.upload.UploadListener
import com.xxl.hello.service.data.repository.DataRepositoryKit
import com.xxl.hello.service.upload.api.UploadService
import java.io.File

/**
 * 腾讯云上传服务实现类
 *
 * @author xxl.
 * @date 2022/5/28.
 */
class TencentUploadServiceImpl(application: Application, dataRepositoryKit: DataRepositoryKit) : UploadService {

    //region: 页面生命周期

    /**
     * 上传
     * @param file
     * @param listener
     */
    override fun upload(file: File,
                        listener: UploadListener) {
        Log.e("aaa", "upload: 我是腾讯云上传${file.absolutePath}")
    }

    //endregion
}